### PR TITLE
Add `context` parameter to the enrichment functions

### PR DESCRIPTION
**Pull request by vlad-lubenskyi**
Adresses #813 

In this PR, we pass `EventContext` as a second parameter to enrichments functions.

Thus:
 - the enrichment functions are not longer `Function<Source, Target>`, but `BiFunction<Source, Context, Target>`;
 - `FieldEnrichment` updated respectively;
 - unit tests updated respectively.

### CODE REVIEW

**Comment by vlad-lubenskyi** [comment]
@armiol PTAL

File: server/src/main/java/io/spine/server/outbus/enrich/FieldEnrichment.java [hunk 1]
```diff
@@ -37,9 +37,10 @@
 final class FieldEnrichment<S, T, C extends Message> extends EnrichmentFunction<S, T, C> {
```
Comment (armiol) [inline server/src/main/java/io/spine/server/outbus/enrich/FieldEnrichment.java]: While you are here, please document the `C` generic parameter.

File: server/src/test/java/io/spine/server/outbus/enrich/EnricherBuilderTest.java [hunk 2]
```diff
@@ -51,15 +52,15 @@
...
     private Enricher.AbstractBuilder builder;
-    private Function<Timestamp, StringValue> function;
+    private BiFunction<Timestamp, Int32Value, StringValue> function;
     private FieldEnrichment<Timestamp, StringValue, ?> fieldEnrichment;
 
     @BeforeEach
     void setUp() {
         this.builder = EventEnricher.newBuilder();
-        this.function = new Function<Timestamp, StringValue>() {
+        this.function = new BiFunction<Timestamp, Int32Value, StringValue>() {
```
Comment (armiol) [inline server/src/test/java/io/spine/server/outbus/enrich/EnricherBuilderTest.java]: Please use the value second parameter type (e.g. `EventContext`) here and below.

Once we get rid of rejections (and they become rejection events), the second parameter will always be `EventContext`; that includes `C` parameter in the `FieldEnrichment` class, which will be removed from the generic params at all.

**Comment by vlad-lubenskyi** [comment]
@armiol PTAL, the last build failed because of dependency fetching timeout

File: ext.gradle [hunk 3]
```diff
@@ -25,7 +25,7 @@
  *  as we want to manage the versions in a single source.
  */
  
-def final SPINE_VERSION = '0.10.75-SNAPSHOT'
+def final SPINE_VERSION = '0.10.76-SNAPSHOT'
```
Comment (armiol) [inline ext.gradle]: Please make sure that your version is an increment — if you have to merge from `master`.

**Pull request merged**